docs/code-of-conduct.md

```diff
@@ -0,0 +1,128 @@
+
+## Unwelcome Behavior
+
+Hard to define? Some questions to ask yourself are:
+
+* how would I feel if I were in the position of the recipient?
+
+* would my spouse, parent, child, sibling or friend like to be treated this way?
+
+* would I like an account of my behavior published in the organization's newsletter?
+
+* could my behavior offend or hurt other members of the work group?
+
+* could someone misinterpret my behavior as intentionally harmful or harassing?
+
+* would I treat my boss or a person I admire at work like that?
+
+_Summary_: if you are unsure whether something might be welcome or unwelcome, don't do it.
+
+## Unwelcome Sexual Advance
+
+includes requests for sexual favors, and other verbal or physical conduct of a sexual nature, where:
+
+* submission to such conduct is made either explicitly or implicitly a term or condition of an individual's employment,
+
+* submission to or rejection of such conduct by an individual is used as a basis for employment decisions affecting the individual,
+
+* such conduct has the purpose or effect of unreasonably interfering with an individual's work performance or creating an intimidating hostile or offensive working environment.
+
+## Workplace Bullying
+
+is a tendency of individuals or groups to use persistent aggressive or unreasonable behavior (e.g. verbal or written abuse, offensive conduct or any interference which undermines or impedes work) against a co-worker or any professional relations.
+
+## Work Environment
+
+is the set of all available means of collaboration, including, but not limited to messages to mailing lists, private correspondence, Web pages, chat channels, phone and video teleconferences, and any kind of face-to-face meetings or discussions.
+
+# Incident Procedure
+
+To report incidents or to appeal reports of incidents, send email to Mike Dolan ([mdolan@linuxfoundation.org](mailto:mdolan@linuxfoundation.org)) or Angela Brown ([angela@linuxfoundation.org](mailto:angela@linuxfoundation.org)). Please include any available relevant information, including links to any publicly accessible material relating to the matter. Every effort will be taken to ensure a safe and collegial environment in which to collaborate on matters relating to the Project. In order to protect the community, the Project reserves the right to take appropriate action, potentially including the removal of an individual from any and all participation in the project. The Project will work towards an equitable resolution in the event of a misunderstanding.
+
+# Credits
+
+This code is based on the [W3C� s Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc "https://www.w3.org/Consortium/cepc") with some additions from the [Cloud Foundry](https://www.cloudfoundry.org/ "https://www.cloudfoundry.org/")� s Code of Conduct.
```

Comment:
W3Câ s -> W3C's

Reply:
â s (exists twice here) and needs to be converted to 's

Reply:
thanks, sharp eyes!